File: addons/point_of_sale/models/uom.py
from odoo import api, fields, models


class UomUom(models.Model):
    _name = 'uom.uom'
    _inherit = ['uom.uom', 'pos.load.mixin']

    is_pos_groupable = fields.Boolean(string='Group Products in POS', help="Check if you want to group products of this unit in point of sale orders")

    @api.model
    def _load_pos_data_fields(self, config):
        taxes = self.env['account.tax'].search(self.env['account.tax']._check_company_domain(config.company_id.id))
        product_uom_fields = taxes._eval_taxes_computation_prepare_product_uom_fields()
        return list(product_uom_fields.union({'id', 'name', 'factor', 'is_pos_groupable', 'parent_path', 'rounding'}))

File: enterprise/stock_barcode_product_expiry/static/tests/mock_models/uom.js
import { fields, models } from "@web/../tests/web_test_helpers";

export class UoM extends models.Model {
    _name = "uom.uom";
    name = fields.Char();
    category_id = fields.Many2one({ relation: "uom.category" });
    factor = fields.Float();
    rounding = fields.Float();
    _records = [
        {
            id: 4,
            name: "Units",
            category_id: 1,
            factor: 1.0,
            rounding: 0.01,
        },
    ];
}
